### PR TITLE
appliance: use Debian 11.8.0 and systemvm version to 4.19.0

### PR DESCRIPTION
**Pull request by rohityadavcloud**
- Bumps ACS systemvmtemplate version
- Bumps systemvmtemplate version used in pom.xml
- Uses Debian 11.8.0 iso

### CODE REVIEW

**Comment by rohityadavcloud** [comment]
@shwstppr cc @weizhouapache could you help merge any other PRs on main first that require new systemvmtemplate (CKS or related)? There are a few interesting security updates but I'm not sure if we need a new template just because of them https://www.debian.org/security/

**Comment by rohityadavcloud** [comment]
Note: build will fail until templates are built and published from this PR to download.cloudstack.org, so let's avoid building/testing anything and revisit this once all other PRs needed systemvmtemplate changes are merged.

**Review by weizhouapache — APPROVED** [review]
great we will have a new systemvm template with latest debian release

**Review by DaanHoogland — APPROVED** [review]

**Comment by shwstppr** [comment]
> @shwstppr cc @weizhouapache could you help merge any other PRs on main first that require new systemvmtemplate (CKS or related)? There are a few interesting security updates but I'm not sure if we need a new template just because of them https://www.debian.org/security/

Thanks @rohityadavcloud. The only other PR that needs systemvm template I can see right now is #7650 which needs some testing cc @weizhouapache

**Comment by weizhouapache** [comment]
> > @shwstppr cc @weizhouapache could you help merge any other PRs on main first that require new systemvmtemplate (CKS or related)? There are a few interesting security updates but I'm not sure if we need a new template just because of them https://www.debian.org/security/
> 
> Thanks @rohityadavcloud. The only other PR that needs systemvm template I can see right now is #7650 which needs some testing cc @weizhouapache

@shwstppr @rohityadavcloud 
the fix requires cloud-init 23.3, which contains the following fix
```
ds-identify/CloudStack: $DS_MAYBE if vm running on vmware/xen (#4281)
```
however, the latest cloud-init version in debian 11 is still 20.4.1. 
let's move the issue #7650 to 4.19.1.0

**Comment by rohityadavcloud** [comment]
Alright thanks @weizhouapache @shwstppr in that case should I go ahead with a systevmtemplate build (or it begs the question - do we even need a new systemvmtemplate as 4.18.1.0 template was only last mont)?

**Comment by weizhouapache** [comment]
> Alright thanks @weizhouapache @shwstppr in that case should I go ahead with a systevmtemplate build (or it begs the question - do we even need a new systemvmtemplate as 4.18.1.0 template was only last mont)?

@rohityadavcloud 
I am +1 with new systemvm template because of
- new debian minor release
- cloudstack bug fixes

File: pom.xml [hunk 1]
```diff
@@ -50,7 +50,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <project.systemvm.template.location>https://download.cloudstack.org/systemvm</project.systemvm.template.location>
-        <project.systemvm.template.version>4.18.1.0</project.systemvm.template.version>
+        <project.systemvm.template.version>4.19.0.0</project.systemvm.template.version>
```
Comment (rohityadavcloud) [inline pom.xml:53]:
```suggestion
        <project.systemvm.template.version>4.18.1.0</project.systemvm.template.version>
```

**Comment by rohityadavcloud** [comment]
@shwstppr could you review and go ahead and merge this? I've reverted the change to enforce the systemvmtemplate version; but this at least fixes the build (iso url/checksum) and bumps the systemvmtemplate version if we decide to build a new one in future.

**Comment by shwstppr** [comment]
@blueorangutan package

File: pom.xml (hunk 1 above)
Comment (rohityadavcloud) [inline pom.xml:53]: Fixed @shwstppr wait before kicking packaging, I need to build/scp the templates to d.c.o server.

**Comment by rohityadavcloud** [comment]
New template uploaded here - https://download.cloudstack.org/systemvm/4.19/ cc @shwstppr @weizhouapache 

@blueorangutan package

**Comment by rohityadavcloud** [comment]
@blueorangutan package

**Comment by weizhouapache** [comment]
@blueorangutan test matrix

**Review by DaanHoogland — APPROVED** [review]
clgtm

**Review by shwstppr — APPROVED** [review]
LGTM. Need to do some testing with the new template

**Review by weizhouapache — APPROVED** [review]
code lgtm
let's wait for the smoke test results

**Comment by rohityadavcloud** [comment]
Keep me posted from your testing @shwstppr @weizhouapache

**Comment by rohityadavcloud** [comment]
@blueorangutan package

**Comment by weizhouapache** [comment]
@blueorangutan package

**Comment by weizhouapache** [comment]
@blueorangutan package

**Comment by rohityadavcloud** [comment]
ping @shwstppr @weizhouapache - if we this want this for 4.19?
@blueorangutan package

**Comment by shwstppr** [comment]
@rohityadavcloud smoke tests look good. I feel we can go ahead and merge this.
@weizhouapache were you able to do any testing?

**Comment by weizhouapache** [comment]
> @rohityadavcloud smoke tests look good. I feel we can go ahead and merge this. 

+1

> @weizhouapache were you able to do any testing?

yes @shwstppr 
I checked the systemvm template of trillian environments, they used the correct template (4.19.0.0) .
I also tested some life cycles of VM and CKS clusters (scaling out/in) manually, all look ok.

**Comment by DaanHoogland** [comment]
I am doing upgrade tests ATM @shwstppr @weizhouapache. I'll report on those A.S.A.P.

**Pull request merged**